web/cobrands/oxfordshire/assets.js

```diff
@@ -480,40 +479,50 @@ var streetlight_select = $.extend({
     fontWeight: 'bold'
 }, fixmystreet.assets.style_default_select.defaultStyle);
 
+function oxfordshire_light(f) {
+    return f && f.attributes && !f.attributes.private;
+}
+function oxfordshire_light_not(f) {
+    return !oxfordshire_light(f);
+}
+
+var light_default_style = new OpenLayers.Style(occ_default);
+var rule_light_owned = new OpenLayers.Rule({
+    filter: new OpenLayers.Filter.FeatureId({
+        type: OpenLayers.Filter.Function,
+        evaluate: oxfordshire_light
+    })
+});
+var rule_light_not_owned = new OpenLayers.Rule({
+    filter: new OpenLayers.Filter.FeatureId({
+        type: OpenLayers.Filter.Function,
+        evaluate: oxfordshire_light_not
+    }),
+    symbolizer: {
+        fillColor: "#868686",
+        pointRadius: 4
```

Comment:
Making the stroke width a bit narrower too (`strokeWidth: 1`) would help visually distinguish these:

<img width="441" alt="image" src="https://user-images.githubusercontent.com/4776/190121377-c0533144-8f47-4630-8a89-3c8fa20a91cd.png">